app/api/models/genome.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import Optional
+
+from pydantic import BaseModel, Field, validator
+
+from core.logging import InterceptHandler
+from core.config import ENA_ASSEMBLY_URL
+
+logging.getLogger().handlers = [InterceptHandler()]
+
+
+class Type(BaseModel):
+    kind: str = Field(alias='kind')
+    value: str = Field(alias='value')
+
+
+class AssemblyInGenome(BaseModel):
+    accession_id: str = Field(alias='accession')
+    name: str = Field(alias='name')
+    url: str = Field(alias='accession', default=None)
+
+    @validator("url", always=True)
+    def generate_url(cls, value):
+        if value and "GCA" in value:
+            return ENA_ASSEMBLY_URL + value
+        return None
+
+
+class AssemblyProvider(BaseModel):
+    name: str = Field(alias='', default=None)  # REMOVE DEFAULT
+    url: str = Field(alias='', default=None)
+
+
+class AnnotationProvider(BaseModel):
+    name: str = Field(alias='annotationProviderName', default=None)  # REMOVE DEFAULT
+    url: str = Field(alias='annotationProviderUrl', default=None)
```

Comment:
Could we treat assembly and annotation providers same way we treat type? That is, if they don't have a name, then the whole field becomes null. If it is allowed for them not to be present in the database, of course.

Reply:
Done!

Reply:
One other thing about the annotation provider. Could you check [with this Slack comment](https://genomes-ebi.slack.com/archives/CC2RKR525/p1698253672974359?thread_ts=1698236783.028169&cid=CC2RKR525) whether we always want to generate a url to rapid when the provider is Ensembl?